plugins/aws/plugin.yaml

```diff
@@ -0,0 +1,1738 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+topology_template:
+  policies:
+    cloudify-aws-plugin:
+      description: >-
+        aws plugin executes operations.
+      type: aria.Plugin
+      properties:
+        version: 1.0
+        enabled: true
```

Comment:
`enabled true` should be the default, please verify it is and if so remove this line